doc/radosgw/admin.rst

```diff
@@ -433,6 +433,8 @@ Usage
 The Ceph Object Gateway logs usage for each user. You can track
 user usage within date ranges too.
 
+- Add ``rgw enable usage log = true`` in [global] section of ceph.conf 
```

Comment:
I guess doesn't need to be in global section... it should be fine in the [client.rgw] ie. rgw section of conf also? Maybe we word it like add ``rgw..` in the relevant rgw section of ceph.conf ?